com.github.Ryujinx.json

```diff
@@ -0,0 +1,1266 @@
+{
+    "app-id": "com.github.Ryujinx",
```

Comment:
The project has a domain so something like `org.ryujinx.Ryujinx` would be a better fit.

Reply:
Yep